modules/logger/src/logger.c

```diff
@@ -45,7 +45,6 @@ static int addJSONString(FILE* fout, const char* jsonString)
         }
         else
         {
-            LogInfo("Message Received to Log: %s", jsonString);
```

Comment:
Hey Damon, this line is removed. 

---
In reply to: [108968038](https://github.com/Azure/azure-iot-gateway-sdk/pull/189#discussion_r108968038) [](ancestors = 108968038)